app/src/main/java/io/apicurio/registry/storage/RegistryStorage.java

```diff
@@ -52,6 +53,14 @@
      */
     public StoredArtifact getArtifact(String artifactId) throws ArtifactNotFoundException, RegistryStorageException;
 
+    /**
+     * Gets the most recent version of the content of the artifact with the given global ID.
+     * @param id
+     * @throws ArtifactNotFoundException
+     * @throws RegistryStorageException
+     */
+    public StoredArtifact getArtifact(long id) throws ArtifactNotFoundException, RegistryStorageException;
```

Comment:
Right, this is required for confluent API support.  But based on my understanding of how our concepts map to theirs, I think this should be `getArtifactVersion(long id)` - because I believe the confluent API in question (as you guys have pointed out to me!) actually gets a specific **version** of a schema.

Reply:
Yeah, each global id maps to exactly one tuple <artifactId, version>.
So I guess this is then a matter of right naming?
(as we use "getArtifactVersion" for a specific tuple, and "getArtifact" for the latest)